snyk-monitor/templates/deployment.yaml

```diff
@@ -166,7 +166,11 @@ spec:
             optional: true
         - name: workload-policies
           configMap:
+            {{- if .Values.defaultWorkloadPolicies.enabled }}
```

Comment:
Would it be possible to flip the order of the if statement, so that instead having the following check `if .Values.defaultWorkloadPolicies.enabled`, check for the existence of workloadPoliciesMap (i.e. check if workloadPoliciesMap is set)?

This also applies to [line 18](https://github.com/snyk/kubernetes-monitor/pull/812/files#diff-037fa431fc058f8bafd63b4732e44bfeb414e6f93fc9dc3c57f5a0ff047c2c71R18) in configmap.yaml, where instead of checking for the enabled flag, we could possibly check if 'workloadPoliciesMap' is not set.

This way we can possibly remove the 'enabled' flag from defaultWorkloadPolicies.

Reply:
ok, I get what you're saying, will try that